packages/polaris-viz/src/components/TooltipWrapper/TooltipWrapper.tsx

```diff
@@ -94,16 +94,16 @@ function TooltipWrapperRaw(props: BaseProps) {
     }) => {
       const containerBounds = {
         x: parentElement?.getBoundingClientRect().x ?? 0,
-        y:
-          Number(parentElement?.getBoundingClientRect().y ?? 0) +
-          Number(scrollContainer?.scrollTop ?? 0),
+        y: Number(parentElement?.getBoundingClientRect().y ?? 0),
+        // + Number(scrollContainer?.scrollTop ?? 0),
```

Comment:
@philschoefer and I added this because `containerBounds` from `useChartContext` wasn't being updated before the long press. I'll see if there's a way around it